main/src/CoffeeMachineSimulatorMain.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+public class CoffeeMachineSimulatorMain {
+
+
+     public static String getCurrentLine;
+
+    //printing auctions, call myMethod
+    public static void main(String[] args) {
+        System.out.println("Write action (buy, fill, take, remaining, exit):");
+        getCurrentLine = getString();
+        CoffeeMachineSimulator coffeeMachineSimulator = new CoffeeMachineSimulator();
+        coffeeMachineSimulator.checkMethod(getCurrentLine);
+
+
+    }
+
+     // method getString read input from console
+    public static String getString() {
```

Comment:
@Edvard777 jan please rename not descriptive

Reply:
done